Orange/widgets/classify/owrandomforest.py

```diff
@@ -19,14 +19,14 @@ class OWRandomForest(widget.OWWidget):
 
     inputs = [("Data", Table, "set_data"),
               ("Preprocessor", Preprocess, "set_preprocessor")]
-    outputs = [("Random Forest", RandomForestLearner),
+    outputs = [("Learner", RandomForestLearner),
                ("Model", RandomForestClassifier)]
 
     want_main_area = False
     resizing_enabled = False
 
     LEARNER = RandomForestLearner
-    learner_name = settings.Setting("RF Classification Learner")
+    learner_name = settings.Setting("RF Classifier")
     n_estimators = settings.Setting(10)
```

Comment:
Well, these names are too long to display nicely in the narrow table headers. How about `'RF Class. Learner'` like you had before? Or perhaps the table needs patching to left-align the headers.